### DISCUSSION TITLE
SSC superadmin inventory wipe

### DISCUSSION THREAD

**Discussion by zaneshaw** [Pryaxis/TShock#2005] [Q&A]
I've been playing with my freinds and I had really nice stuff on the server. The fountains and campfires were not working for default group so i went into superadmin to turn them on/off.
It said `skipping SSC backup for "username"` in the console and when I went back to default, all of my items were GONE! Is there a way I can rollback my world or ssc inv?

**Comment by hakusaro** [comment]
Hmm, maybe temporary elevation to superadmin is what did it? That's awkward.
